### DISCUSSION TITLE
Can I block YouTube as a source?

### DISCUSSION THREAD

**Discussion by tbmedia** [jagrosh/MusicBot#1125] [Questions]
So just recently I've received DMCA's from YouTube, and also had my IP blocked... I think they unblocked me, though, and that's a problem since my friends on Discord may play more music from there and give me **more** DMCA's. When I get enough of these, they shut my internet off permanently.

So, can I disable YouTube as a source?

**Comment by MichailiK** [comment]
Disabling a source is not possible. Though, you might be able to set up a Firewall to prevent JMusicBot from connecting to Google/YouTube servers.
